src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -230,6 +277,7 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
         }
 
         // Check if the motion between the nearest state and the state to add is valid
+        ++collisionChecks_;
```

Comment:
I don't think it is meaningful. Worse, people might think it literally means number of collision checks, which it doesn't. In other words, I think you should remove it.